new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItInitContainers.java

```diff
@@ -0,0 +1,443 @@
+
+    //check if init container got executed
+    assertTrue(assertDoesNotThrow(() -> getPodLog(domain1Uid + "-admin-server", domain1Namespace,"busybox")
+            .contains("Hi from Domain"),
+        "failed to init busybox container command for admin server"));
+    assertTrue(assertDoesNotThrow(() -> getPodLog(domain1Uid + "-managed-server1", domain1Namespace,"busybox")
+            .contains("Hi from Domain"),
+        "failed to init busybox container command for managed server1"));
+    assertTrue(assertDoesNotThrow(() -> getPodLog(domain1Uid + "-managed-server2", domain1Namespace,"busybox")
+            .contains("Hi from Domain"),
+        "failed to init busybox container command for managed server2"));
+
+  }
+
+  /**
+   * Add initContainers to adminServer and verify the admin server pod executes initContainer command
+   * and starts the admin server pod.
+   * Test fails if domain crd can't add the initContainers or
+   * weblogic server pod doesn't go through initialization and ready state
+   */
+  @Test
+  @DisplayName("Add initContainers to adminServer and verify the admin server pod executes initContainer command ")
+  public void testAdminServerInitContainer() {
+    assertTrue(createVerifyDomain(domain2Namespace, domain2Uid, "adminServer"),
+        "can't start or verify domain in namespace " + domain2Namespace);
+
+    //check if init container got executed for admin server pod
+    assertTrue(assertDoesNotThrow(() -> getPodLog(domain2Uid + "-admin-server", domain2Namespace,"busybox")
+            .contains("Hi from AdminServer"),
+        "failed to init busybox container command for admin server"));
+  }
+
+  /**
+   * Add initContainers to adminServer and verify the managed server pods in cluster execute initContainer command
+   * before starting the admin server pod.
+   * Test fails if it cannot include the initContainers or
+   *                   weblogic server pod doesn't go through initialization and ready state
+   */
+  @Test
+  @DisplayName("Add initContainers to cluster1 and verify all managed server pods go through Init state ")
+  public void testClusterInitContainer() {
+    assertTrue(createVerifyDomain(domain3Namespace, domain3Uid, "clusters"),
+        "can't start or verify domain in namespace " + domain3Namespace);
+
+    //check if init container got executed
+    assertTrue(assertDoesNotThrow(() -> getPodLog(domain3Uid + "-managed-server1",
+        domain3Namespace,"busybox").contains("Hi from Cluster"),
+        "failed to init busybox container command for cluster's managed-server1"));
+    assertTrue(assertDoesNotThrow(() -> getPodLog(domain3Uid + "-managed-server2",
+        domain3Namespace,"busybox").contains("Hi from Cluster"),
+        "failed to init busybox container command for cluster's managed-server2"));
+  }
+
+  /**
+   * Add initContainers to managed-server1 and verify managed server pod executes initContainer command
+   * before starting the admin server pod.
```

Comment:
before starting the admin server pod or managed-server1 pod?

Reply:
done